### ISSUE TITLE
fix code formatting (help wanted from community)

### ISSUE THREAD

**Issue by elahehrashedi** [microsoft/vscode-cmake-tools#1883]
A reminder to ask the community to help us fix the code formatting, 
including indentation, wrapping the lines, ...

**Comment by bobbrow** [comment]
I don't think this is something we'll ask the community for.  We should set up a formatting profile and just format the files using the VS Code Format File command.  I've been meaning to do it, but the last time we got down to 0 PR's we got a whole bunch of new ones before I had a chance to do it. 😄

**Comment by elahehrashedi** [comment]
TODO:
- search for all  "tslint:disable ..." in the code.
- some rules are commented out in .eslintrc.js, bring them back one by one.
